platform/ui/src/views/Viewer/Viewer.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
```

Comment:
Same as `StudyList.js`, I don't think we will need this component to be exposed, or even to exist.

Reply:
I agree, but before we have in on '@ohif/viewer' I am totally fine by having it here, so we can test it out of the docz.

Reply:
I'm ok with that. My only concern is that we will have to go through it later to delete those files that are left behind. Perhaps they will not be many. Let's think about it later.